src/services/telemetryService.js

```diff
@@ -92,21 +92,21 @@ export class TelemetryService {
         .map(toMessagesModel);
   }
 
-  static getTelemetryByDeviceIdP1M(devices = []) {
+  static getTelemetryByDeviceIdP1M(Devices = []) {
     return TelemetryService.getTelemetryByMessages({
-      from: 'NOW-PT1M',
-      to: 'NOW',
-      order: 'desc',
-      devices
+      From: 'NOW-PT1M',
+      To: 'NOW',
+      Order: 'desc',
+      Devices
```

Comment:
If you want the API to receive 'Devices', just do this -> `Devices: devices`. You don't have to change the variable name everywhere.

1. These methods been working well already correct? 
2. why this change now? 
3. How is this related to KeyVault stuff? 

If this is unrelated I'd recommend creating a separate PR.